src/main/java/org/spongepowered/api/entity/player/Player.java

```diff
@@ -79,8 +82,11 @@
     Locale getLocale();
 
     /**
-     * Sends the plain text message(s) with the specified {@link ChatType} on the client.
-     * <p>Use {@link #sendMessage(ChatType, Message...)} for a formatted message.</p>
```

Comment:
@AlphaModder I usually select only the code I've changed and press `Ctrl+Shift+F`, that will format the code selected and no other.
I also use a feature of git-gui that allows me to stage only certain lines of code, and ignore the rest ([see here](http://stackoverflow.com/questions/1085162/commit-only-part-of-a-file-in-git) for how to do this on the cmd line)